federated-identity/src/test/java/com/iluwatar/federatedIdentity/federatedIdentityTest.java

```diff
@@ -0,0 +1,42 @@
+package com.iluwatar.federatedIdentity;
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+public class federatedIdentityTest {
+    Consumer consumer;
+    IdP idP;
+    Server server;
+
+    @Before
+    public void init(){
+        consumer = new Consumer();
+        idP = new IdP();
+        server = new Server();
+        idP.listener();
+        server.listener();
+    }
+
+    @Test
+    public void test1(){
```

Comment:
Please use meaningful names for the tests so it's easy to see at a glance what it does